### ISSUE TITLE
v1.78.0 of VSCode imposes max-height of CodeTour step window

### ISSUE THREAD

**Issue by kazemicode** [microsoft/codetour#285]
v1.78.0, specifically, [this PR](https://github.com/microsoft/vscode/pull/180044), imposes a strict max-height for the UI component that CodeTour steps are displayed in. For anyone who uses CodeTour to embed images/video, this means that any media that surpasses this max-height cannot be seen in the frame all at once. I've proposed a [feature request](https://github.com/microsoft/vscode/issues/181846) to expose functionality via the VSCode API to set the max-height rather than having an imposed max-height. At this time, it's unclear what the default behavior will be. If the default behavior goes back to dynamically adjusting the height of the UI component with the length of the content, no changes are warranted on the CodeTour side. If, however, the default behavior stays as it is now (a default max-height), we will need to specify a larger max-height value for CodeTour windows to display media in a non-cumbersome way.

**Comment by Sayan4444** [comment]
Is there any progress for this particular PR?

**Comment by kazemicode** [comment]
@Sayan4444 yes. This issue should be closed! There is now a user setting comments.maxHeight that you can set to False. Take a look at the thread in the feature request I linked to in this post.

**Comment by Sayan4444** [comment]
@kazemicode That is so helpful. Thank you
